test/e2e/specs/all-features-confirmation/recordedit/add.spec.js

```diff
@@ -212,10 +221,43 @@ describe('Record Add', function() {
                 }
             });
         });
-
+        describe('Markdown Editor Help button is clicked, ', function() {
```

Comment:
On the second thought, it's better to bring it outside the outer `describe`. It's currently inside "When url has a prefill query string param set," which shouldn't be. Then you can open the recordedit page yourself and don't need to add those assumptions as it will be clearly visible from code that it's going to which app and table.